website/docs/reference/node-selection/yaml-selectors.md

```diff
@@ -112,6 +140,8 @@ where we can only pass one "yeslist" (`--select`) and one "nolist" (`--exclude`)
 
 #### Indirect selection
```

Comment:
i know this already exists but do you think there's value in making these (+ the exclude h4 header) into H3 headers?

Reply:
Yeah, would be great to take a look at upgrading the headers for the whole page.